source/diagrams/navigate-diagrams/working-with-entities/delete-entities.txt

```diff
@@ -1,3 +1,63 @@
+.. _rm-delete-entities:
+
 ===============
 Delete Entities
-===============
\ No newline at end of file
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Select one or more entities to delete from your Relational or MongoDB 
+model. Deleting entities allows you to:
+
+- Customize the size of your diagram.
+- Focus on entities you select.
+- Visualize your relational database's schema to help determine how to best 
+  represent your data in MongoDB. 
+
+About this Task
+---------------
+
+- You can revert any deletion. 
+
+- Deleting an entity on the :guilabel:`Relational Model` is the equivalent of removing a 
+  table from the project's model. 
+
```

Comment:
(nit)
For linking, you can either link inline, like in the third bullet. But then you would add sentence cased link text. (In this case all lower case.) 

Or, add a See :ref: Title. statement. Then it's fine to leave title casing. 

Holler at me if that doesn't make sense.